src/main/java/com/ldtteam/blockout/controls/TextField.java

```diff
@@ -626,6 +663,8 @@ public void deleteFromCursor(final int count)
                 this.moveCursorBy(count);
             }
         }
+
+        triggerHandler();
```

Comment:
Might probably be double call because of line 602

Reply:
Should be if/else there too, I tried it out and it would only trigger once for me